gef-ui/server/utils/validateFile.js

```diff
@@ -1,6 +1,8 @@
-const DEFAULT_MAX_SIZE = 1024 * 1024 * 10;
-const DEFAULT_ALLOWED_FORMATS = ['bmp', 'doc', 'docx', 'gif', 'jpeg', 'jpg', 'msg', 'pdf', 'png', 'ppt', 'pptx', 'tif', 'txt', 'xls', 'xlsx', 'zip'];
+const filesize = require('filesize');
 
+const DEFAULT_MAX_SIZE = 10; // 10mb
+const DEFAULT_UNITS = ['KiB', 'B', 'kbit'];
+const DEFAULT_ALLOWED_FORMATS = ['bmp', 'doc', 'docx', 'gif', 'jpeg', 'jpg', 'msg', 'pdf', 'png', 'ppt', 'pptx', 'tif', 'txt', 'xls', 'xlsx', 'zip'];
```

Comment:
Brink of max-character counter, please lint as following:

[
'bmp',
'doc','
'docx'
]